src/Shared/Data/Database/Skills.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Aura development team - Licensed under GNU GPL
+// For more information, see license file in the main folder
```

Comment:
Skill and Skills cause confusion. Does melia also use SkillManager like Aura or is there no such system present?